bin/sgqlc-codegen

```diff
@@ -326,30 +334,36 @@ default=%(default)s)),
 
     def write_type_input_object(self, t):
```

Comment:
nit: `write_type_input_object` and `write_type_interface` are almost exactly the same you may want to move some of their common logic to a function.

Reply:
done, check the updated PR.

Reply:
Good!